resource/ui/ts/models/timeseries.ts

```diff
@@ -1,5 +1,6 @@
 // source: models/timeseries.ts
```

Comment:
Yeah there needs to be a rename of one of these, i'm always hesitant to do it while changing the file because it results in unreadable diffs.